content/tutorial/tutorial.md

```diff
@@ -421,17 +426,18 @@ Instead, we'll pass down a function from the Board to the Square, and we'll have
   }
 ```
 
->Note
+>ध्यान दें
 >
->We split the returned element into multiple lines for readability, and added parentheses so that JavaScript doesn't insert a semicolon after `return` and break our code.
+>हमने पठनीयता के लिए दिए गए एलिमेंट को कई लाइनों में विभाजित किया है, और कोष्ठक को जोड़ा है ताकि जावास्क्रिप्ट `return` के बाद सेमीकोलन न डालें और हमारे कोड को तोड़ दें।
```

Comment:
कोष्ठक -> to complex

Reply:
`return` के बाद सेमीकोलन न डालें और हमारे कोड को तोड़ दें। changes the meaning of the original sentence

Reply:
> कोष्ठक -> to complex

Let keep it `पैरेंथेसेस`?

Reply:
> > कोष्ठक -> to complex
> 
> Let keep it `पैरेंथेसेस` or `ब्रैकेट्स`?

works

Reply:
> और हमारे कोड को तोड़ दें

Any suggestions for this?

Reply:
code ko chote hisson main baant den?